classes/wrapper-stage.oeclass

```diff
@@ -0,0 +1,59 @@
+## Class for packages depending on recipes having wrapper files.
+##
+## Rewrites the paths in the wrapper files so that it contains valid dirs.
+
+require conf/meta.conf
+
+STAGE_FIXUP_FUNCS += "wrapper_stage_fixup"
+
+WRAPPER_STAGE_DIRNAMES = "prefix exec_prefix bindir sbindir libdir \
+	includedir libexecdir datarootdir datadir sysconfdir sharedstatedir \
+	localstatedir infodir mandir"
+
+# FIXME: add a method for packages providing wrapper files to specify which
+# dirnames to fixup, so that fx. this installbuilddir could be provided by apr
+# recipe, so that it does not impact metadata, and thus signature of all
+# recipes.
+WRAPPER_STAGE_DIRNAMES += "installbuilddir"
```

Comment:
Hehe.

But you only removed the FIXME comment.  What about the installbuilddir append, couldn't you just remove that and do it in the apr recipe instead?